packages/deployer/docker/README.md

```diff
@@ -1,130 +1,19 @@
-# api3/deployer
+# Deployer docker image
 
-**This documentation focuses on the Deployer Docker image and its usage, not the Deployer itself. If you want learn more
-about the Deployer, please read [its documentation](../README.md).**
+**This README is intended for the developers. Documentation on how to use the deployer image is in the
+[docs](https://docs.api3.org/airnode/next/grp-providers/docker/deployer-image.html).**
 
-## Build
-
-In order to build Deployer Docker image you need to build the [artifacts image first](../../../docker/README.md). Once
-you've done that, you can build the Docker image by running following command from the root directory:
-
-```bash
-docker build -f packages/deployer/docker/Dockerfile -t api3/deployer:latest .
-```
-
-## Configuration
-
-### Credentials
+This documentation focuses on the Deployer docker image, not the Deployer itself. If you want learn more about the
+Deployer, please read [its documentation](../README.md).
 
-In order to deploy Airnode to a cloud provider like AWS, you need to provide your cloud credentials to the container.
```

Comment:
THe rest of the README looks like "usage" to me and should be ported to the api3 docs (cc @wkande this is probably what you mean on adding, right?)